cmd/omegaup-grader/frontend_handler.go

```diff
@@ -320,6 +327,36 @@ func broadcastRun(
 	if err != nil {
 		return err
 	}
+	var groupName string
+	var scoreByGroup float64
+	rows, error := queryWithRetry(
+		db,
+		`SELECT
+			rg.group_name, rg.score
+		FROM
+			Runs_Groups rg
+		WHERE
+			rg.run_id = ?;`, run.ID)
+	if error != nil {
+		return error
+	}
+
+	var scores map[string]float64
+	scores = make(map[string]float64)
```

Comment:
```suggestion
	scores := make(map[string]float64)
```

en general hay que evitar declarar algo sin inicializarlo.

Reply:
Listo.